addons/gunbag/CfgVehicles.hpp

```diff
@@ -10,6 +10,13 @@ class CfgVehicles {
                     showDisabled = 0;
                     icon = QPATHTOF(ui\gunbag_icon_ca.paa);
                 };
+                class GVAR(weaponSwap) {
+                    displayName = CSTRING(SwapGunbag);
+                    condition = QUOTE(([_target] call FUNC(hasGunbag)) && {[ARR_2(_player,_target)] call FUNC(canInteract) == 2} && (GVAR(swapGunbagEnabled)));
```

Comment:
Put simple `GVAR` condition first and lazy eval rest.

Reply:
I'm more of a C# / java guy than a C++ guy so I haven't dealt with header files to much extent before. Can I do this with a standard if statement before the class definition or are you implying some other way that I should implement this?